README.md

```diff
@@ -1,5 +1,28 @@
 # Eden
 
-Eden is where [EVE](https://github.com/lf-edge/eve) and [Adam](https://github.com/lf-edge/eve) get tried and tested.
+Eden is where [EVE](https://github.com/lf-edge/eve) and [Adam](https://github.com/lf-edge/adam) get tried and tested.
 
 Eden consists of a test harness and a series of integration tests implemented in Golang. Tests are structured as normal Golang tests by using ```_test.go``` nomenclature and be available for test runs using standard go test framework.
+
+## Install Dependencies
+
+Install requirements from [eve](https://github.com/lf-edge/eve#install-dependencies)
+
+Also, you need to install ``uuidgen```
+
```

Comment:
Please fix quotes -- you can even use single backticks here @sadov @giggsoff

Reply:
Fixed